src/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Project Le News</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <div class="container1" id="container">
+        <div class="header" id="head">
+            <div class="header__logo" id="figureLogo">
+                <figure id="Logo">
+                    <img src="images/LeNewsLogo.png" alt="Logo of Le News" />
+                </figure>
+            </div>
+
+            <div class="header__search" id="formSearch">
+                <form class="search__Searchbar" action="action_page.php">
+                    <input type="text" placeholder="Search..." name="search">
+                    <button type="submit">
+                        <img class="q" src="images/searchLogo.png" alt="logo of a magnifying glass">
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <div class="box1" id="article1">
+            <h1 class="box1--position-left" id="title1">
+                <a target="_self" href="images/LeNews.drawio.html">
+                    Title</a>
+            </h1>
+            <div class="box1--position-left box1--color-grey" id="date category">Date of Publication, <br> Category
+            </div>
+            <div class="box1--position-right box1--color-grey" id="teaser1">
+                Lorem ipsum dolor sit amet, sapien etiam, nunc amet dolor ac odio mauris justo.
+                Luctus arcu, urna praesent at id quisque ac. Arcu es massa vestibulum malesuada, integer vivamus elit eu
+                mauris eus, cum eros quis aliquam wisi. Nulla wisi laoreet suspendisse integer vivamus elit eu mauris
+                hendrerit facilisi, mi mattis pariatur aliquam pharetra eget.
+            </div>
+        </div>
+
+        <div class="container2">
+            <div class="container2__box2" id="article2">
+                <h2 id="title3">
+                    <a target="_self" href="images/LeNews.drawio.html">
+                        Title
+                    </a>
+                </h2>
+                <br>
+                <div class="container2__box2__teaser" id="teaser2">
+                    Lorem ipsum dolor sit amet, sapien etiam, nunc amet dolor ac odio mauris justo.
+                    Luctus arcu, urna praesent at id quisque ac. Arcu es massa vestibulum malesuada, integer vivamus
+                    elit eu
+                    mauris eus, cum eros quis aliquam wisi. Nulla wisi laoreet suspendisse integer vivamus elit eu
+                    mauris
+                    hendrerit facilisi, mi mattis pariatur aliquam pharetra eget.
+                </div>
+            </div>
+
+            <div class="container2__box2" id="article3">
+                <h2 id="title3">
```

Comment:
Aaaaaaand here it is: the number one reason why people dont like IDs .. this id is already in use. My proposal: use classes everywhere anytime.

Reply:
oh a really bad mistake. I deleted every IDs